setup.py

```diff
@@ -59,16 +57,21 @@
 setup(
     name="aac",
     version="0.0.4",
+    description=(
+        "A distinctly different take on Model-Based System Engineering (MBSE) that allows a system modeller to define a system in simple yaml. "
+    ),
+    license="MIT",
```

Comment:
Did we for sure decide to license it with the MIT license, then?

Reply:
It's already the Jellyfish's license. I can remove the license for now.
https://github.com/NorthropGrumman/jellyfish/blob/master/LICENSE

Reply:
```suggestion
    license="",
```